app-i18n/mozc/files/mozc-2.23.2815.102-protobuf-3.18.patch

```diff
@@ -0,0 +1,11 @@
+--- /src/dictionary/user_dictionary_storage.cc
++++ /src/dictionary/user_dictionary_storage.cc
```

Comment:
please use the git patch style, like the rest of us (this applies to all other patches too)

Reply:
+1, I'd also appreciate if you could provide commit messages and keep the author/signed-off information when doing git-format-patch